packages/vx-axis/src/utils/getTickPosition.ts

```diff
@@ -0,0 +1,24 @@
+import { ScaleInput } from '@vx/scale';
+import { AxisScale, AxisScaleOutput } from '../types';
+
+/**
+ * Create a function that returns a tick position for the given tick value
+ */
+export default function getTickPosition<Scale extends AxisScale>(scale: Scale) {
+  // Broaden type before using 'xxx' in s as typeguard.
+  const s = scale as AxisScale;
+
+  // For point or band scales,
+  // have to add offset to make the tick centered.
```

Comment:
Is there a use case for making this configurable?  `(start | center | end)`

Reply:
Adding additional alignment would be nice addition. Will mod it.
(The original file was called `center`.)